site/docs/components/table.mdx

```diff
@@ -38,10 +41,18 @@ HDS Tables are currently only available for HDS Core and they include base style
 - Table container is allowed to be scrollable. This is common on smaller screens where it can be impossible to fit the full table on the device screen.
   - On smaller screens, it may be beneficial to consider other presentation formats for table data. For example, a table can be broken into cards.
 
+### Interactable tables (React)
+- HDS Tables support sorting. When a column is set to be sortable, a sorting button appears after its header.
```

Comment:
Single column cannot be set sortable; it's all columns or no sorting at all at the moment

Reply:
Hmm. I do not know whether this is a good thing or not. In audit, we were told to think carefully when to use sorting. Now it is easily possible that some of the columns are not easily sortable when you mostly want to sort names/number etc.

Reply:
- [ ] Clarified how table sorting works in the table docs